src/common/components/DialogWithForm.tsx

```diff
@@ -1,12 +1,14 @@
 import { Form, FormProps, Modal, ModalProps } from "antd";
 import { FormInstance } from "antd/es/form/Form";
-import React, { ReactNode } from "react";
+import React, { ReactNode, useEffect } from "react";
+import { showOverlay, removeOverlay } from "../../pageServices/pageDataHandlers";
 
 interface JDNModalProps extends ModalProps {
   setIsModalOpen: (value: boolean) => void;
   open: boolean;
   onOk: () => void;
   cancelCallback?: () => void;
+  enableOverlay?: boolean;
```

Comment:
This solution has restrictions: if we will use this component without conditional rendering, it will lead to bugs. Most likely, we will newer face with such case, so, please add a comment about this restriction here, it will be enough for now